feature-libs/asm/assets/translations/en/asm.ts

```diff
@@ -26,6 +26,12 @@ export const asm = {
       submit: 'Start Session',
       noMatch: 'No customer found.',
     },
+    bindCart: {
+      cartNumber: 'Cart Number',
+      bindCartToCustomer: 'Assign Cart to Customer',
+      success: 'Cart has been successfully assigned',
+      error: 'Cart failed to be assigned',
```

Comment:
this error message is generic. Can we catch the error and display the error message sent from backend?